src/System.Windows.Forms/src/System/Windows/Forms/ListView.cs

```diff
@@ -6931,14 +6931,13 @@ protected override void WndProc(ref Message m)
                 case User32.WM.KEYUP:
                     int key = (int)m.WParamInternal;
 
-                    if ((key == User32.VK.LEFT || key == User32.VK.RIGHT) && SelectedItems.Count > 0)
+                    if (GroupsDisplayed && (key == User32.VK.LEFT || key == User32.VK.RIGHT) && SelectedItems.Count > 0)
```

Comment:
```suggestion
                    if (GroupsDisplayed && (key is User32.VK.LEFT or User32.VK.RIGHT) && SelectedItems.Count > 0)
```

Reply:
```suggestion
                    if (GroupsDisplayed && (key == User32.VK.LEFT || key == User32.VK.RIGHT) && SelectedItems.Any())
```
Though I'm not sure about performance, https://stackoverflow.com/a/63569609 (see .NET Core).

Reply:
Hi @Tanya-Solyanik, @dreddy-work, @RussKie. Could you please share your opinion on which is better to use `SelectedItems.Count > 0` or `SelectedItems.Any()`.

Reply:
Not so sure but Count would seem fine here.